cpp/zotero_harvester.cc

```diff
@@ -55,7 +55,9 @@ using namespace ZoteroHarvester;
               << "\tSelection modes: UPLOAD, URL, JOURNAL\n"
               << "\t\tUPLOAD - Only those journals that have the specified upload operation (either LIVE or TEST) set will be processed.\n"
               << "\t\tURL - Only the specified URL is processed as a DIRECT harvester operation.\n"
-              << "\t\tJOURNAL - If no arguments are provided, all journals are processed. Otherwise, only those journals specified are processed.\n\n";
+              << "\t\tJOURNAL - If no arguments are provided, all journals are processed. Otherwise, only those journals specified are processed.\n"
+              << "\t\t          If \"--config-overrides=\" are specified, it will be treated as raw text to override parts of the config file sections.\n"
```

Comment:
The "--config-overrides" help text should be moved above with the rest of the optional parameters. The overrides should apply to all the selection modes, in my opinion. The current implementation will automatically take apply to the UPLOAD and JOURNAL modes as we're applying the overrides at the config parsing stage. The URL mode, however, will need more work. 

My suggestion is that we just mention in the help text that only UPLOAD and JOURNAL selection modes are affected by the overrides.